src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -190,30 +202,25 @@ export default {
     > li button,
     > li span,
     > :nth-child(n + 2)::before {
-      color: var(--oc-color-text-muted);
-    }
-    > :nth-child(n + 2)::before {
-      content: "/";
```

Comment:
Why not use the `>` char here?

Reply:
in figma its an icon https://www.figma.com/file/EqAiB1wY7cYaMaitjyzup3/Current-oC-Web-UI-Kit-(WIP)?node-id=272%3A99266

Reply:
Okay, tested it and doesn't look even remotely the same. I'll give the full icon aria-hidden though since it's not relevant for screenreaders